src/main/java/uk/ac/ebi/eva/pipeline/io/writers/AnnotationMongoWriter.java

```diff
@@ -118,10 +117,10 @@ private void writeAnnotationInMongoDb(BulkOperations bulk, Annotation annotation
     }
 
     private BasicUpdate buildUpdateQuery(Annotation annotation) {
-        final BasicDBObject addToSetValue = new BasicDBObject();
+        final Document addToSetValue = new Document();
         addToSetValue.append(CONSEQUENCE_TYPE_FIELD, buildInsertConsequenceTypeQuery(annotation));
         addToSetValue.append(XREFS_FIELD, buildInsertXrefsQuery(annotation));
-        return new BasicUpdate(new BasicDBObject(ADD_TO_SET, addToSetValue));
+        return new BasicUpdate(new Document(ADD_TO_SET, addToSetValue));
     }
 
     private BasicDBObject buildInsertXrefsQuery(Annotation annotation) {
```

Comment:
Any reason why this method does not return Document instead of BasicDBObject?

Reply:
No reason, I just missed that one. It is changed now